ydb/core/protos/config.proto

```diff
@@ -1517,10 +1517,24 @@ message TCompactionConfig {
 }
 
 message TTracingConfig {
+    message TTvmCredentials {
+        required uint32 SelfTvmId = 1;
+        required uint32 TracingTvmId = 2;
+
+        optional string DiskCacheDir = 3;
+
+        oneof Secret {
+            string PlainTextSecret = 4;
+            string SecretFile = 5;
+            string SecretEnvironmentVariable = 6;
+        }
+    }
+
     optional string Host = 1;
     optional uint32 Port = 2;
     optional string RootCA = 3;
     optional string ServiceName = 4;
+    optional TTvmCredentials TvmCredentials = 5;
```

Comment:
I think it would be better to make generic message Auth or Authentication, and then put TvmCredentials inside into it. also it's not clear when we expect to use Tvm for authentication. when some fields are not empty?

Reply:
At the moment we have [`AuthConfig`](https://github.com/ydb-platform/ydb/pull/768/files#diff-2e781bd76ab9922cc0012cb685cbf5adcc215185dd89fb3b9f2566f407c62f3bR1520) with the only auth option being Tvm